### ISSUE TITLE
Privacy Missing Heading and Landmark Regions

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/MathShare#786]
The main content of the Privacy page is not contained inside a < main> ARIA landmark region, nor is it preceded by a semantically accessible heading.  Similarly, the Benetech logo image is not contained inside a < header> region. 
 
Suggested Solution 
 
Mark up the title text (“Privacy”) as a < h1> heading.  Then wrap the content between the Benetech logo header image and start of the footer inside a < main> element.  The logo itself should be wrapped inside a <header> (or < div> with a role of “banner”).

**Comment by jscholes** [comment]
Fixed in https://github.com/benetech/MathShare/commit/225ba29b88a062d15679c01d52c3fbbe4898a9c7.
